deployment/ibm_cloud_pak_for_security/deploy.sh

```diff
@@ -2,6 +2,8 @@
 
 cd "$(dirname "$0")"
 
+PYTHONIOENCODING='utf8'
```

Comment:
This never gets referenced?

Reply:
it used by python https://docs.python.org/3/using/cmdline.html#envvar-PYTHONIOENCODING